src/components/entity/EntityList.vue

```diff
@@ -116,13 +116,24 @@
                             <div class="row" v-if="baseUrl === 'item'">
                                 <div class="col-6 text-left form-group">
                                     <label>External Source</label>
-                                    <input
+                                    <!-- <input
                                         type="text"
                                         class="form-control w-100"
                                         v-model="entity.externalSource"
                                         :disabled="showEdit"
                                         @change="onInputChange"
-                                    />
+                                    /> -->
+                                    <select
+                                        class="select custom-select w-100"
+                                        v-model="entity.externalSource"
+                                        :class="{ 'error-border': (submitted && !entity.externalSource) }"
+                                        @change="onInputChange"
+                                    >
+                                        <option
+                                            v-for="option in listOfExternalResources"
+                                            :key="option"
+                                        >{{ option }}</option>
+                                    </select>
                                 </div>
```

Comment:
externalSource is for Item and Collection only. Will this change in EntityList impact other entities?

Reply:
So far we have displayed external source dropdown only for an Item, and Yes it is a conditional rendering element. we have condition on line no 116